package.json

```diff
@@ -15,6 +15,8 @@
     "lib/*"
   ],
   "dependencies": {
+    "@sentry/node": "^6.2.5",
+    "@sentry/tracing": "^6.2.5",
```

Comment:
If we can move everything into Brave Alert Lib, then we won't need to do this explicitly, it'll be there automatically as a dependency of `brave-alert-lib`. Like how I was able to remove the direct `dotenv` dependency from Buttons and Sensors.